src/app/collections/trip_list.js

```diff
@@ -0,0 +1,44 @@
+import Backbone from 'backbone';
+import Trip from '../models/trip';
+
+const TripList = Backbone.Collection.extend({
+  model: Trip,
+  url: 'https://ada-backtrek-api.herokuapp.com/trips',
+  comparator: 'id',
+  greaterThan(field, value) {
+    let models = this.select(function(model) {
+      return model.get(`${field}`) > value;
+    });
+    return new TripList(models);
+  },
+  includes(field, value) {
+    if (field === undefined || field === '') {
+      let models = this.select(function(model) {
+        const attributes = Object.keys(model.attributes);
+
+        for (let i = 0; i < attributes.length; i += 1) {
+          if (typeof model.get(attributes[i]) === 'number') {
+            if (model.get(attributes[i]).toString().includes(value)) return true;
+            // return model.get(attributes[i]).toString().includes(value);
+          } else {
+            if (model.get(attributes[i]).toLowerCase().includes(value.toLowerCase())) return true;
```

Comment:
This code section could be simplified a bit by having each attribute always get `toString()` called on it (which is unnecessary but harmless for the string attributes):
```javascript
const val = value.toLowerCase();

for (let i = 0; i < attributes.length; i++) {
  const attr = model.get(attributes[i]).toString().toLowerCase();
  if (attr.includes(val)) return true;
}
```